cpp/daal/src/externals/service_rng_mkl.h

```diff
@@ -33,6 +33,7 @@
 #define __DAAL_BRNG_MT19937                   VSL_BRNG_MT19937
 #define __DAAL_BRNG_MCG59                     VSL_BRNG_MCG59
 #define __DAAL_RNG_METHOD_UNIFORM_STD         VSL_RNG_METHOD_UNIFORM_STD
+#define __DAAL_RNG_METHOD_UNIFORMBITS32_STD   0
```

Comment:
Why 0?

Reply:
VSL_RNG_METHOD_UNIFORMBITS32_STD (value is 0) don't exist in mkl fpk, so, it's workaround